src/Sylius/Bundle/CoreBundle/Resources/config/services/listeners/workflow.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would vote to split this config file into
> listeners/workflow/order.xml
> listeners/workflow/order_checkout.xml
> listeners/workflow/order_payment.xml
> listeners/workflow/order_shipping.xml

with 
```xml
   <imports>
        <import resource="listeners/workflows/*.xml" />
    </imports>
```

Reply:
LGTM :)